compliance/collection/metrics/metrics.go

```diff
@@ -0,0 +1,141 @@
+package metrics
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stackrox/rox/generated/internalapi/sensor"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/metrics"
+)
+
+var (
+	numberOfRHELPackages = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.ComplianceSubsystem.String(),
+		Name:      "packages_in_inventory",
+		Help:      "Number of packages discovered by the last Node Inventory (per Node)",
+	},
+		[]string{
+			// The Node this scan belongs to
+			"node_name",
+			// The OS name and version of the Node
+			"os_namespace",
+		})
+
+	numberOfContentSets = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.ComplianceSubsystem.String(),
+		Name:      "content_sets_in_inventory",
+		Help:      "Number of content sets discovered by the last Node Inventory (per Node)",
+	},
+		[]string{
+			// The Node this scan belongs to
+			"node_name",
+			// The OS name and version of the Node
+			"os_namespace",
+		})
+
+	scanDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
In the cluster I see that 100% of scans take between 2.5 and 5 seconds (query:`rox_compliance_inventory_scan_duration_seconds_bucket{node_name="mm-01-27-scientific-f-cvdq4-worker-b-ndhlf"})`. This is long. Could we double check that the values are correct (especially seconds vs milliseconds error should be avoided)?

Reply:
I've checked the log messages in a collector, and it looks correct. We definitely need to  keep an eye on how this develops. Multiple seconds for what we're doing today seems rather long-ish.

```
$ ks logs collector-ss48q compliance | grep collecting
collection/nodeinventorizer: 2023/01/30 10:05:13.789081 nodeinventory.go:27: Debug: Start collecting node inventory
collection/nodeinventorizer: 2023/01/30 10:05:18.750015 nodeinventory.go:37: Debug: Finished collecting node inventory
```

Reply:
After updating to a better debug log message, I can confirm the observation on an OpenShift Node:
`Debug: Collecting Node Inventory took 2.882744 seconds`